applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -887,18 +997,15 @@ static void endpoint_cb(struct bt_conn *conn, enum bt_audio_dir dir, struct bt_b
 
 	if (dir == BT_AUDIO_DIR_SINK) {
 		if (ep != NULL) {
-			if (unicast_server->num_sink_eps > 0) {
-				LOG_WRN("More than one sink endpoint found, idx 0 is used "
-					"by default");
-				return;
-			}
-
-			unicast_server->sink_ep = ep;
+			/* Since we need to discover all sink endpoints first, we store them in a
+			 * temporary array in the first unicast server, to be distributed later
+			 */
+			unicast_server->spare_sink_eps[unicast_server->num_sink_eps] = ep;
```

Comment:
Not sure I understand this. Regardless of mono or stereo, we will only ever need a 1 or 2 sink endpoint. Why do we want to store all of them?